README.md

```diff
@@ -474,13 +482,26 @@ function getTrackerLoader(reactiveMapper) {
   };
 }
 
+function composeWithTracker(loadFunc) {
```

Comment:
Out of curiosity - why not? Doesn't seem to break the pattern and offers a drop-in replacement for v1 users...